projects/nae-example-app/src/app/doc/public-single-task-view/public-single-task-view.component.html

```diff
@@ -0,0 +1,33 @@
+<div class="task-tab-background full-height">
+    <div fxLayout="column" fxLayoutAlign="start stretch" class="content-margin full-height" >
+        <nc-header [hidden]='hidden' #header type="task" class="task-panel-padding-mini"></nc-header>
+        <nc-single-task [task$]="task$" [loading$]="loading$" [selectedHeaders$]="selectedHeaders$" (taskEvent)="logEvent($event)"
+                        [actionRowContent]='actionRowContent' [pageHeader]='pageHeader' [pageFooter]='pageFooter'
+                        [actionRowJustifyContent] = '"center"' fxFlex></nc-single-task>
+    </div>
+</div>
+
+<ng-template #actionRowContent>
+    <div class="public-button-padding">
+        <button mat-raised-button color="primary" class="public-button"
+                (click)="finish();">
+            <span>{{getFinishTitle()}}</span>
+        </button>
+    </div>
+</ng-template>
+
+<ng-template #pageHeader>
+    <div>
+        <img src="../../../assets/img/netgrif_full_white.svg" alt="logo" class="logoimg">
+        <mat-divider class="upper-divider"></mat-divider>
+    </div>
+    <mat-card class="upper-card">
+        <span class="upper-card-title"><strong>Example header</strong></span>
+    </mat-card>
+</ng-template>
+
+<ng-template #pageFooter>
+    <mat-toolbar color="primary" class="footer-custom">
+        <span>&copy; 2022 NAE s.r.o.</span>
```

Comment:
NAE s.r.o.? :rofl:

Reply:
That was mistake from my side, I just copied a simple template from web and rewrote the company title to NAE, now it is corrected to Netgrif.